view/topbar.twig

```diff
@@ -15,7 +15,7 @@
 <div id="language"><span class="navigation-font">|</span>
   {% for langcode, langdata in languages %}
   {% if request.lang != langcode %}
-  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("#{request.lang}/"): "#{langcode}/"}) }}"> {{langdata.name}}</a>
+  <a id="language-{{ langcode}}" class="navigation-font" href="{{ request.langurl|replace({("#{request.lang}/"): "#{langcode}/"}) }}"> {{langdata.lemma}}</a>
```

Comment:
The problem you are seeing (with **in_this_language** displayed) is caused by missing locales on your system. See the [FAQ](https://github.com/NatLibFi/Skosmos/wiki/FAQ-and-Troubleshooting). Please revert this part.